.eslintrc.js

```diff
@@ -0,0 +1,84 @@
+module.exports = {
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    tsconfigDir: __dirname,
+    project: "./tsconfig.json",
```

Comment:
are these options needed when the tsconfig is at the root of the project. i would have thought that's the default values.

Reply:
Unfortunately yes otherwise you get an error like:
 > `Error: Error while loading rule '@typescript-eslint/prefer-nullish-coalescing': You have used a rule which requires parserServices to be generated. You must therefore provide a value for the "parserOptions.project" property for @typescript-eslint/parser.
Occurred while linting /Users/....../Web/vue-test-utils-next/src/vueWrapper.ts`